script/Arcueid/Day 8/08_00_ARC08_2B.txt

```diff
@@ -196,7 +197,7 @@ But he doesn't smell like bourgeois at all, does he?!"
 [sha:d22831132d02817cc2bdb11c371bc4670e7152e4]{
 -- Page 107, Offset 19937.
 -- 「どんな匂いだソレ。金に匂いなんてねえだろ。それとも手垢の匂いってコトか？　金持ちはドイツもコイツも垢だらけってか？　ずいぶん舐めた発言だなオイ？」
-"What kind of smell is that? It's not like you can smell riches right? Or are you talking about the smell of finger marks? Are you saying that the rich are nothing but dirt? That's rather rich coming from you, oi."
+"What kind of smell is that? It's not like you can smell money right? Or are you talking about the smell of dirt from his hands? Are you saying that the rich are nothing but filth? That's rather rich coming from you, oi."
```

Comment:
I'm not entirely sure what 手垢 is referring to but this is the word used for like the grime left behind by your fingers on stuff you touch often. I thought the first time I read this that it was referring to the lingering presence of money, so that would need to be reconciled too.
> Or are you talking about the smell of the filth on his hands? Are you saying that's all the wealthy are? Filth? That's rather rich coming from you, oi.